.github/workflows/au4_build_linux.yml

```
@@ -276,8 +276,120 @@ jobs:
+  testflow-discover:
+    # Lists the test scripts so that testflow-linux can fan out one job per
+    # script. Reads the repo (same commit the AppImage is built from) instead
+    # of the artifact, so it runs in parallel with the build.
+    runs-on: ubuntu-22.04
+    outputs:
+      scripts: ${{ steps.list.outputs.scripts }}
+    steps:
+    - name: Clone repository
+      uses: actions/checkout@v6
+      with:
+        repository: ${{ inputs.app_repo || job.workflow_repository }}
```

actionlint (1.7.12)

[error] property "workflow_repository" is not defined in object type {check_run_id: number; container: {id: string; network: string}; services: {string => {id: string; network: string; ports: {string => string}}}; status: string}

(expression)